net.sourceforge.roccat.roccat-tools.yml

```diff
@@ -0,0 +1,66 @@
+app-id: net.sourceforge.roccat.roccat-tools
+runtime: org.gnome.Platform
+sdk:  org.gnome.Sdk
+runtime-version: '44'
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --device=all
+  - --device=dri
+  # necessary for audio feedback
+  # - --socket=pulseaudio
+  # - --socket=session-bus
+rename-icon: roccat
+
+modules:
+  - shared-modules/lua5.4/lua-5.4.json
+  - shared-modules/libcanberra/libcanberra.json
+  - shared-modules/gtk2/gtk2.json
+  - shared-modules/dbus-glib/dbus-glib.json
+
+  - name: libgaminggear
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_C_FLAGS=-isystem /usr/include/harfbuzz
+      - -DINSTALL_PKGCONFIGDIR=/app/lib/pkgconfig
+      - -DINSTALL_CMAKE_MODULESDIR=/app/lib/cmake
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DINSTALL_LIBDIR=/app/lib
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/libgaminggear/libgaminggear-0.15.1.tar.bz2
+        sha1: 78cf02ae89cb0e4d8276cfef7da78a49c7c83a44
+
+  - name: roccat-tools
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_C_FLAGS=-isystem /usr/include/harfbuzz
+      - -DINSTALL_PKGCONFIGDIR=/app/lib/pkgconfig
+      - -DINSTALL_CMAKE_MODULESDIR=/app/lib/cmake
+      - -DCMAKE_MODULE_PATH=/app/lib/cmake
```

Comment:
I don't think these 3 lines are necessary....

Reply:
When I remove `DINSTALL_PKGCONFIGDIR=/app/lib/pkgconfig` I get 
```sh
CMake Error at configuration/cmake_install.cmake:54 (file):
  file INSTALL cannot copy file
  "/run/build/libgaminggear/configuration/gaminggear.pc" to
  "/usr/share/pkgconfig/gaminggear-0.pc": Read-only file system.
```

Reply:
I remove `DINSTALL_PKGCONFIGDIR` and `DINSTALL_CMAKE_MODULESDIR`, but `DCMAKE_MODULE_PATH` seemd to still be required.